src/components/BrandConfigProvider/theme/ThemeProvider.tsx

```diff
@@ -9,17 +9,8 @@ import {
   adaptNavigationTheme,
   Provider as PaperProvider,
 } from 'react-native-paper';
-import * as baseDefaultTheme from './base/default';
-
-// Merges baseDefaultTheme and reactNavigationDefault into MD3DefaultTheme
-// which is the default theme returned by usePaperTheme
-const combinedDefaultTheme = merge(
-  MD3DefaultTheme,
-  reactNavigationDefault,
-  baseDefaultTheme,
-);
```

Comment:
This was mutating MD3DefaultTheme in an effort to create a combined type.  Among other things, it caused our tests to accidentally work without mocking.  I added the useTheme mock and typing instead.